python/docs/AclProfileApi.md

```diff
@@ -0,0 +1,1060 @@
+# semp_client.AclProfileApi
+
+All URIs are relative to *http://localhost/SEMP/v2/config*
```

Comment:
Why are you keeping docs in GitHub? These should be generated as part of the build process.